### ISSUE TITLE
Getting an error when leaving a space between `doctype` and `html`

### ISSUE THREAD

**Issue by lobo-tuerto** [slime-lang/slime#160]
Hello,
I'm just trying out Slime and while converting my `app.html.eex` file I found out that if I write this:

```slim
doctype html

html
```

I get an error like:

```text
Compiling 1 file (.ex)

== Compilation error in file lib/rumbl_web/views/layout_view.ex ==
** (Slime.TemplateSyntaxError) Unexpected symbol '
'
INPUT, Line 2, Column 1
```

Notice the empty line between `doctype` and `html`.
If I remove the space everything is ok.

It also works like this:

```slim
doctype html
|
html
```

Is there any particular reason why an empty line is not allowed between the `doctype` and `html` tags?

**Comment by Rakoth** [comment]
Hi, @lobo-tuerto, thanks for the report, sounds like a bug to me. Would you like to created a PR and try to fix it?

**Comment by lobo-tuerto** [comment]
@Rakoth Sure, I'll take a look and take a stab at it. :)

**Comment by lobo-tuerto** [comment]
@Rakoth Got a PR for this issue ready to be reviewed here: https://github.com/slime-lang/slime/pull/161 :)

**Comment by Rakoth** [comment]
Fixed by #161
